composer/datasets/in_context_learning_evaluation.py

```diff
@@ -892,6 +892,7 @@ def __init__(
         destination_path: str,
         code_prelimiter: str,
         fewshot_random_seed: int,
+        pass_at_k: int,
```

Comment:
Can we default to 1 here so that this is not API breaking?

Reply:
addressed